frontstage/controllers/party_controller.py

```diff
@@ -109,6 +109,20 @@ def get_respondent_by_email(email):
     return response.json()
 
 
+def resend_verification_email(party_id):
+    logger.debug('Re-sending verification email', party_id=party_id)
```

Comment:
IMHO 'info' means "useful information when the service is operating normally" and 'debug' means "extra information which might help with debugging, but is otherwise not useful when service is robust and trusted". I'd say that any happy-path logging is debug, not info. Just my two cents tho

Reply:
That sounds like a compelling argument, I'm happy to go with this. I always get confused 😕